src/include/miopen/generic_search.hpp

```diff
@@ -343,6 +346,13 @@ auto GenericSearch(const Solver s, const Context& context, const AnyInvokeParams
     HeartBeat<PerformanceConfig> heartbeat;
     heartbeat.Start();
 
+    const char* const c_and_r = miopen::GetStringEnv(MIOPEN_COMPILE_AND_RUN{});
+    std::string compile_and_run;
+    if(c_and_r != nullptr && strlen(c_and_r) > 0)
+    {
+        compile_and_run = c_and_r;
+    }
+
```

Comment:
This is not necessary.

Reply:
Which part? We need to check this env var to skip invokers launching kernels.